be/src/io/writable.h

```diff
@@ -0,0 +1,36 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#pragma once
+
+#include "common/status.h"
+
+namespace starrocks::io {
+
+class Writable {
+public:
+    virtual ~Writable() = default;
+
+    // Write the given data to the stream
+    //
+    // This method always processes the bytes in full. Depending on the
+    // semantics of the stream, the data may be written out immediately,
+    // held in a buffer, or written asynchronously. In the case where
+    // the stream buffers the data, it will be copied.
+    virtual Status write(const void* data, int64_t size) = 0;
+
+    virtual bool allows_aliasing() const = 0;
```

Comment:
What does this function mean?

Reply:
> What does this function mean?

Whether the writable support `write_aliased`